sdk/src/main/java/io/dapr/actors/runtime/ActorRuntime.java

```diff
@@ -97,7 +99,7 @@ public static ActorRuntime getInstance() {
 
     ActorFactory actualActorFactory = actorFactory != null ? actorFactory : new DefaultActorFactory<T>(actorTypeInfo);
       // TODO: Refactor into a Builder class.
-      DaprStateAsyncProvider stateProvider = new DaprStateAsyncProvider(this.appToDaprAsyncClient, new ActorStateSerializer());
+      DaprStateAsyncProvider stateProvider = new DaprStateAsyncProvider(this.appToDaprAsyncClient, new ObjectSerializer());
```

Comment:
I made this change in a different way, you can use the ObjectSerializer but the ActorStateSerializer remains too for Actor-specific stuff.

Reply:
https://github.com/dapr/java-sdk/pull/62